packages/openapi-generator/src/cli/index.ts

```diff
@@ -40,6 +40,14 @@ class GenerateOpenApiClient extends Command {
       default: false,
       required: false
     }),
+    strictNaming: flags.boolean({
+      name: 'strictNaming',
+      description:
+        'By default, the generation fails, when there are duplicate names for operations and/or path parameters after transforming them to camel case. Set this to true to enable unique name generation. The names will then be generated by appending numbers.',
+      default: true,
+      required: false,
+      allowNo: true
```

Comment:
I guess oclif will add additional doc when `allowNo` is true? If not, we might want to mention it.

Reply:
Good point. I just checked and this is what it shows on `--help`:

```sh
--[no-]strictNaming                          By default, the generation fails, when there are duplicate names for operations and/or path parameters after transforming them to camel case. Set this to true to enable unique name generation. The names will 
                                               then be generated by appending numbers.
```

Good enough, right?